infrastructure/tooling/src/generate/generators/docker-compose.js

```diff
@@ -297,16 +308,40 @@ exports.tasks.push({
       },
     };
 
+    ['standalone', 'static'].forEach(type => {
+      dockerCompose.services[`generic-worker-${type}`] = serviceDefinition('generic-worker', {
+        image: 'taskcluster/generic-worker:local', // TODO build and publish this image as well?
```

Comment:
Could we remove this todo and maybe make a github issue for it if you see fit?

Reply:
Good point! Yes, I'll create a ticket for it

Reply:
https://github.com/taskcluster/taskcluster/issues/5583